api/users_test.go

```diff
@@ -25,6 +25,9 @@ const usersEndPointWithActiveFilterTrue = "http://localhost:25600/v1/users?activ
 const usersEndPointWithActiveFilterFalse = "http://localhost:25600/v1/users?active=false"
 const usersEndPointWithActiveFilterError = "http://localhost:25600/v1/user?active=false"
 
+const usersEndPointWithSortByEmail = "http://localhost:25600/v1/users?sort=email"
+const usersEndPointWithSortByEmailAcs = "http://localhost:25600/v1/users?sort=email:asc"
```

Comment:
```suggestion
const usersEndPointWithSortByEmailAsc = "http://localhost:25600/v1/users?sort=email:asc"
```

Reply:
:asc is not mandatory however :desc is mandatory 
 I am testing both